awscognito/cognito_users.py

```diff
@@ -0,0 +1,171 @@
+# pylint: disable=global-statement,redefined-outer-name
+""" Script used to create|disable AWS Cognito user """
+import argparse
+import sys
+from dataclasses import dataclass
+
+import boto3
+import pandas
+import yaml
+
+
+@dataclass
```

Comment:
Use ‘frozen=True’ to indicate it is immutable.

Reply:
Added as well.